napari/plugins/pypi.py

```diff
@@ -0,0 +1,62 @@
+"""
```

Comment:
I'm not 100% sure about this. I guess there is a long term plan to make an "add_plugin" dialog from the GUI that would maybe show the list of available plugins on PyPi, but in that case do we might also want to surface plugins that also don't follow the naming convention, or are on some curated list etc.

I don't really object to this going in now, but might be good to here a bit more about the plan moving forward for this code, as I think it could probably wait till we're a bit closer to having it all integrated. Curious what @jni and @kne42 think?

Reply:
yeah, this would be used for that sort of add plugin thing.  can definitely be removed/delayed if anyone prefers to leave it out for now.

Reply:
I just verified that it works - I can see all the `pytest-*` packages. One thing is though, I don't see any `napari-*` packages even though I just put up an empty `napari-segmentation` which I can pip install. Does it just take some time to update the list it pulls from? Probably not unreasonable that it may take a bit of time to refresh, but would be nice to verify that works too

Reply:
it won't appear in `pypi.get_pypi_packages_by_prefix('napari-')` until your package appears on https://pypi.org/simple/, which doesn't seem to be updated with your package.  However, directly accessing it with `pypi.get_package_versions('napari-segmentation')` does return '(0.0.0,)'

Reply:
I can confirm this now works

Reply:
great.  I made a legitimate reader plugin here: https://github.com/tlambert03/napari-dv
just uploading to pypi now so you can give it a try.

Reply:
Love it, I am very excited about the napari cookiecutter :-)